lib/git_reflow/git_server/pull_request.rb

```diff
@@ -35,13 +53,48 @@ def reviewers_pending_response
         reviewers - approvals
       end
 
+      def check_approvals
```

Comment:
I think this needs a better name, like `approved?` for this.  Since it returns a boolean, at least end it with `?`.